src/Core/Scheduler/Providers/3dTiles_Provider.js

```diff
@@ -48,8 +52,14 @@ $3dTiles_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(/
 
 function getBox(boundingVolume) {
     if (boundingVolume.region) {
-        throw new Error('Region bounding volume not supported yet');
-        // return { region: boundingVolume.region };
+        const region = boundingVolume.region;
+        const radToDeg = 180 / Math.PI;
+        const extent = new Extent('EPSG:4326', region[0] * radToDeg, region[2] * radToDeg, region[1] * radToDeg, region[3] * radToDeg);
```

Comment:
That's actually annoying, since we would like to use 3d tiles for projected data, and avoid re-projections as much as possible. Could we implement support for any projection in iTowns, and try to get this support in the standard afterwards ?